### DISCUSSION TITLE
Wazuh 4.xx question

### DISCUSSION THREAD

**Discussion by Yoyu88** [Security-Onion-Solutions/securityonion#6079] [General] [answered]
Is Wazuh 4.xx working yet with SO 2.3.80 yet? Has anyone upgraded to the current version yet?  If not, what is the highest level of Wazuh that is working with SO and is stable?

**Comment by dougburks** [comment, marked as answer]
Security Onion 2.3.80 includes Wazuh 3.13.1 and that's the only version of Wazuh that we support right now.
